utilities/ideascale-importer/ideascale_importer/tests/test_data/config.json

```diff
@@ -0,0 +1,14 @@
+{  
+    "event_id": 0,
+    "group_id": 37429,
+    "review_stage_ids": [171],
+    "nr_allocations": [1, 1],
+    "campaign_group_id": 88,
+    "questions": {
+        "Question 1": "Impact / Alignment",
+        "Question 2": "Feasibility",
+        "Question 3": "Auditability"
+    },
+    "stage_ids": [4684, 4685, 4686],
+    "anonymize_start_id": 5000
```

Comment:
I believe that we can come up with a better name for the field "anonymize_start_id".
Could you clarify the purpose of `anonymize_start_id`? I'm not entirely clear on what it does.

Reply:
It is id to start with to assign anonymized IDs